docs/source/quickinstall.rst

```diff
@@ -22,11 +22,19 @@ you have `conda` installed on your system, OpenMC can be installed via the
 
     conda config --add channels conda-forge
 
+To list the versions of OpenMC that are available on `conda-forge` channel to install, in your terminal window or an Anaconda Prompt, run:
+
+.. code-block:: sh 
+
+    conda search openmc
+    
 OpenMC can then be installed with:
 
 .. code-block:: sh
 
-    conda install openmc
+    conda install openmc=0.X.0 python=3.X
+    
+.. note:: Replace ``python=3.X`` with ``python=2.7`` if using legacy Python2. ``openmc=0.X.0`` is the package and version you want to install.
```

Comment:
I think a better approach is to recommend creating a new environment, which will avoid clashes with packages installed in another environment. Also, I don't think we should be mentioning Python 2.7 since it is already deprecated at this point:
```suggestion
    conda install -n openmc-env openmc
    
This will install OpenMC in a conda environment called `openmc-env`. To activate
the environment, run:

.. code-block:: sh

    conda activate openmc-env
```